packages/web-react/src/components/FileUploader/FileUploaderContext.ts

```diff
@@ -0,0 +1,18 @@
+import { createContext, useContext } from 'react';
+import { FileUploaderHandlingProps } from '../../types';
+
+export interface FileUploaderContextProps extends FileUploaderHandlingProps {}
+
+const defaultContext: FileUploaderContextProps = {
+  addToQueue: () => null,
```

Comment:
Maybe, it will be great, if the function returns the current state/content of the queue.

Reply:
done